packages/core/src/scss/themeV2/form/_form.scss

```diff
@@ -30,4 +30,32 @@
     //    color: #3c4043;
     //    font-weight: bold;
     //}
```

Comment:
Not in this PR, but can this be removed?

Reply:
L3 can be removed along with the commented code as well.